torchinductor/triton_ops/autotune.py

```diff
@@ -147,13 +176,28 @@ def pointwise_heuristics(size_hints):
         return autotune(
             [
                 triton_config(size_hints, 64, 64),
-                triton_config(size_hints, 4, 256),
-                triton_config(size_hints, 256, 4),
+                triton_config(size_hints, 8, 256),
+                triton_config(size_hints, 256, 8),
+                triton_config(size_hints, 1, 1024),
+                triton_config(size_hints, 1024, 1),
             ],
             key=["xnumel", "ynumel"],
         )
     if len(size_hints) == 3:
-        return apply_triton_config(triton_config(size_hints, 16, 16, 16))
+        if not config.triton.autotune:
+            return apply_triton_config(triton_config(size_hints, 16, 16, 16))
+        return autotune(
+            [
+                triton_config(size_hints, 16, 16, 16),
```

Comment:
these look pretty suspicious lol.

Reply:
What would you suggest?

Reply:
Getting size_hints == 3 should be pretty unusual, do we know under which conditions we are getting it?

Reply:
`config.max_tiles=2` right now, so never.

I was experimenting a bit with `max_tiles=3` (3D) tiling, for the case where you read from 3 different tensors each with different stride=1 dimensions